activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/TaskVariableUpdatedEventHandler.java

```diff
@@ -31,9 +31,12 @@ public TaskVariableUpdatedEventHandler(TaskVariableUpdater variableUpdater) {
     public void handle(TaskVariableEntity updatedVariableEntity) {
         String variableName = updatedVariableEntity.getName();
         String taskId = updatedVariableEntity.getTaskId();
-        BooleanExpression predicate = QTaskVariableEntity.taskVariableEntity.name.eq(variableName)
+        BooleanExpression predicate = QTaskVariableEntity.taskVariableEntity.name.eq(String.valueOf(variableName))
```

Comment:
Are you sure this extra `String.valueOf` is necessary here? `variableName` is already a String.